build/config/mac/mac_sdk.gni

```diff
@@ -13,17 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Utility script to genertae the python tag + the abi tag + platform tag for a Python
+_mac_deployment_target = "10.15"
 
-try:
-  from setuptools import pep425tags
-except ImportError:
-  from wheel import pep425tags
+if (current_cpu == "arm64") {
+  _mac_deployment_target = "11.0"
+}
 
-try:
-  platform_tag = pep425tags.get_platform(None)
-except TypeError:
-  platform_tag = pep425tags.get_platform()
+declare_args() {
+  # OS X version to target when compiling.
+  mac_deployment_target = _mac_deployment_target
```

Comment:
could we grab this from our current python environment?

For me, the output of:

```
python -c "import platform; print(platform.platform().split('-')[1])"
```

is what I would like to use for MACOSX_DEPLOYMENT_TARGET

Reply:
Why would we do that?

Reply:
Deployment target is the minimum OS X version the binary should be supported by. It is a function of CHIP's requirements, not of your running system.